SwrveSDKCommon/Common/SwrveAssetsManager.m

```diff
@@ -0,0 +1,124 @@
+#import <CommonCrypto/CommonHMAC.h>
+#import "SwrveAssetsManager.h"
+#import "SwrveCommon.h"
+
+@implementation SwrveAssetsManager
+
+@synthesize restClient;
+@synthesize assetsCurrentlyDownloading;
+@synthesize cdnRoot;
+@synthesize cacheFolder;
+@synthesize assetsOnDisk;
+
+- (id)initWithRestClient:(SwrveRESTClient *)swrveRESTClient andCacheFolder:(NSString *)cache{
+    self = [super init];
+    if (self) {
+        [self setRestClient:swrveRESTClient];
+        self.cdnRoot = @"https://content-cdn.swrve.com/messaging/message_image/"; // default
```

Comment:
If we do we would need to save the latest CDN as served by content.swrve.com, not a quick change